valohai_cli/commands/pipeline/run/run.py

```diff
@@ -0,0 +1,60 @@
+import click
+
+from valohai_cli.api import request
+from valohai_cli.commands.pipeline.run.utils import build_edges, build_nodes, match_pipeline
+from valohai_cli.ctx import get_project
+from valohai_cli.messages import success, error
+
+
+@click.command(
+    context_settings=dict(ignore_unknown_options=True),
+    add_help_option=False
+)
+@click.argument('name', required=False, metavar='PIPELINE-NAME')
+@click.option('--commit', '-c', default=None, metavar='SHA', help='The commit to use. Defaults to the current HEAD.')
+@click.option('--title', '-c', default=None, metavar='SHA', help='The optional title of the pipeline run.')
```

Comment:
Metavar is not SHA here; leave it out :)
```suggestion
@click.option('--title', '-c', default=None, help='The optional title of the pipeline run.')
```